starter-code/js/index.js

```diff
@@ -1,52 +1,110 @@
-function deleteItem(e){
+window.onload = function () {
+    //////Products
 
-}
+    var calcPriceButton = document.getElementById("calculate-price-btn");
+    var total = document.getElementsByClassName("total")[0];
+    var basketValue = document.getElementById("total-sum");
+    var price = document.getElementsByClassName("price")[0];
+    var deleteButton = document.getElementsByClassName("btn-delete")[0];
+    var creatingNewBtn = document.getElementById("create");
 
-function getPriceByProduct(itemNode){
```

Comment:
There are a few lines wrong within this function. 

Please compare it to the function that i wrote from line 10 to 28 in the following repl > https://repl.it/@MattHamers2/lab-dom-ironhack-cart 

We can go over it during the next class.